.github/workflows/e2e-k3d.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     env:
       ALL_BROWSERS: ${{ github.ref == 'refs/heads/main' || github.event.inputs.all_browsers && 'true' || 'false' }}
       sha: ${{ github.event.pull_request.head.sha || github.sha }}
-      wait_timeout: ${{ github.ref == 'refs/heads/main' && 900 || 240 }}
+      wait_timeout: ${{ github.ref == 'refs/heads/main' && 900 || 600 }}
```

Comment:
How come tests now take twice as long?

Reply:
It takes a little longer due to creating the MinIO instance I think.